diagnostic.md

```diff
@@ -5,20 +5,26 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    A tree can be a visual representation of components, this can show how
+    components can be nested within other components, and how they relate to
+    outer routes, etc.
```

Comment:
What do you mean by a tree?  Can you expand?  A few UI elements; a list that contains list-items, a navbar w/ buttons, or a sidebar for a blog w/ comments